ChatQuickstart/QuickstartChatManager.swift

```diff
@@ -0,0 +1,152 @@
+//
+//  QuickstartChatManager.swift
+//  ChatQuickstart
+//
+//  Created by Jeffrey Linwood on 3/11/20.
+//  Copyright © 2020 Twilio, Inc. All rights reserved.
+//
+
+import UIKit
+
+import TwilioChatClient
+
+protocol QuickstartChatManagerDelegate {
+    func reloadMessages()
+    func receivedNewMessage()
+}
+
+class QuickstartChatManager:NSObject, TwilioChatClientDelegate {
+        
+    // the unique name of the channel you create
+    private let uniqueChannelName = "general"
+    private let friendlyChannelName = "General Channel"
+    
+    // For the quickstart, this will be the view controller
+    var chatManagerDelegate: QuickstartChatManagerDelegate?
+    
+    // MARK: Chat variables
+    private var client: TwilioChatClient? = nil
+    private var channel: TCHChannel? = nil
+    private(set) var messages: [TCHMessage] = []
+    
+    func chatClient(_ client: TwilioChatClient, synchronizationStatusUpdated status: TCHClientSynchronizationStatus) {
+        guard status == .completed else {
+            return
+        }
+        checkChannelCreation { (created) in
+            if created {
+                self.joinChannel()
+            } else {
+                self.createChannel() {
+                    self.joinChannel()
+                }
+            }
+        }
+        
+    }
+    
+    // Called whenever a channel we've joined receives a new message
+    func chatClient(_ client: TwilioChatClient, channel: TCHChannel,
+                    messageAdded message: TCHMessage) {
+        messages.append(message)
+        
+        DispatchQueue.main.async() {
+            if let delegate = self.chatManagerDelegate {
+                delegate.reloadMessages()
+                if self.messages.count > 0 {
+                    delegate.receivedNewMessage()
+                }
+            }
+        }
+    }
+    
+    func sendMessage(_ messageText:String, completion: @escaping (TCHResult, TCHMessage?) -> Void) {
```

Comment:
Keep styles consistent, use space after column.

Reply:
I'm not 100% sure what you meant with this, but I did split the arguments into two lines to minimize line length.

Reply:
In your function declaration you wrote `messageText:String` without space after the column. It is better to add it here and in other places.

Reply:
I went ahead and ran swiftlint on the whole code base, and resolved a number of code cleanup and style issues.